src/scenic/core/object_types.py

```diff
@@ -283,6 +284,28 @@ class Point(_Constructible):
 	def visibleRegion(self):
 		return CircularRegion(self.position, self.visibleDistance)
 
+	def getModifiedVisibilityRegion(self, other, actors):
+		modifiedVisibilityRegion = self.visibleRegion
+		for actor in actors:
+			if actor != self and actor != other:
+				#TODO: If blocked visibility cone is not fully contained by visible region, 
+				# reduce visible region instead, this will create a side-effect to this 
```

Comment:
What side-effect are you referring to?

Reply:
I took care of this but forgot to update the comment!